### PR TITLE
add gcbm installation instructions

### LINKED ISSUE #39: Add GCBM installation documentation
**What does the current documentation state?**
Add GCBM installation documentation

### PR DESCRIPTION
**Pull request by Tlazypanda**
# Pull Request Template

## Description

Add GCBM installation instructions.

Fixes #39 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

### CODE REVIEW

**Comment by aornugent** [comment]
Looking good Sneha. I've added a couple of comments, but otherwise approved.

**Comment by Tlazypanda** [comment]
Hey @aornugent I can't see the comments am I missing something or are they in some other place? :sweat_smile:

**Comment by aornugent** [comment]
> Hey @aornugent I can't see the comments am I missing something or are they in some other place? 😅

My bad, I didn't realise I had to submit my reviews!

File: docs/GCBMDevelopmentSetup/prerequisites.rst [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+You should be familiar with the following:
+
+Python 3.7
+----------
+
+This document will guide you through the steps for installing the Python 3.7 environment required for running the pre- and post-processing tools for GCBM.
+
+If you already have Python installed on your computer, follow the instructions in the section `Existing Python Installation <existing_python_installation.html>`_ .
+
+If Python is not already installed, follow the instructions in the section `New Python Installation <new_python_installation.html>`_ .
```
[the remark this answers is not in the document]
Comment (Tlazypanda) [reply docs/GCBMDevelopmentSetup/prerequisites.rst]: Resolved

File: docs/GCBMDevelopmentSetup/index.rst [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+The recommended method for installing the GCBM development environment is on Windows using Python 3.7 environment.
+
+Contents:
+
+.. toctree::
+  :maxdepth: 1
+
+  prerequisites
+  windows_installation
```
[the remark this answers is not in the document]
Comment (Tlazypanda) [reply docs/GCBMDevelopmentSetup/index.rst:18]: Hey @aornugent actually these files are not public and can be requested for installation so have added that as a line now in index.rst

**Comment by Tlazypanda** [comment]
Hey @aornugent is this good to merge now? 😅

**Comment by aornugent** [comment]
I think it's great! 👏👏

**Pull request merged**